pentaho-js/src/main/javascript/cdf/components/AnalyzerComponent.js

```diff
@@ -57,6 +57,12 @@ define([
       $.map(this.parameters, function(k) {
         options[k[0]] = k.length == 3 ? k[2] : d.getParameterValue(k[1]);
       });
+      for (var option in options){
+          if (Date.parse(options[option])){
+              // see http://jira.pentaho.com/browse/CDF-887
+              options[option] = options[option] + " 00:00:00.0";
```

Comment:
@mbatchelor this pull request has been reverted and a new fix has been developed. The right pr is #880